cardano-crypto-praos/src/Cardano/Crypto/VRF/Praos.hs

```diff
@@ -294,7 +291,7 @@ instance ToCBOR VerKey where
     encodedSizeExpr (\_ -> fromIntegral verKeySizeVRF) (Proxy :: Proxy ByteString)
 
 instance FromCBOR VerKey where
-  fromCBOR = vkFromBytes <$> fromCBOR
+  fromCBOR = fromCBOR >>= either fail return . vkFromBytes
```

Comment:
Oh, we should not be using custom instances here at all anyway. It should be:
```suggestion
  fromCBOR = decodeVerKeyVRF
```
and similarly for all the other `ToCBOR` and `FromCBOR` instances. See the pattern in e.g. `Cardano.Crypto.VRF.Simple`

Reply:
I see I was wrong here. These types should not have To/FromCBOR instances at all, and the exported types should have these custom instances rather than using newtype deriving.

Reply:
I suggest if we believe these instances should be removed, that I submit a new PR for the removal.